src/Mvc/Mvc.Testing/src/WebApplicationFactory.cs

```diff
@@ -159,6 +159,7 @@ private void EnsureServer()
             if (builder is null)
             {
                 var deferredHostBuilder = new DeferredHostBuilder();
+                deferredHostBuilder.UseEnvironment(Environments.Development);
```

Comment:
What's this about?

Reply:
This is the bug being fixed. We do it in the other 2 cases but not in this one 
1. https://github.com/dotnet/aspnetcore/blob/18a926850f7374248e687ee64390e7f10514403f/src/Mvc/Mvc.Testing/src/WebApplicationFactory.cs#L384
2. https://github.com/dotnet/aspnetcore/blob/18a926850f7374248e687ee64390e7f10514403f/src/Mvc/Mvc.Testing/src/WebApplicationFactory.cs#L403